resolver/resolver.cc

```diff
@@ -1483,7 +1483,19 @@ class ResolveTypeMembersAndFieldsWalk {
         auto owner = data->owner;
 
         core::LambdaParam *parentType = nullptr;
-        auto parentMember = owner.data(ctx)->superClass().data(ctx)->findMember(ctx, data->name);
+        core::SymbolRef parentMember = core::Symbols::noSymbol();
+        parentMember = owner.data(ctx)->superClass().data(ctx)->findMember(ctx, data->name);
```

Comment:
OOC, why do we not traverse the entire inheritance hierarchy here?  Is it not available at this point, or have we just decided that's How Generics Work?

Reply:
It works inductively: the parent being marked as resolved depends on all of its parents being marked resolved, so we only need to depend on direct ancestors.